openwisp_radius/base/models.py

```diff
@@ -1139,6 +1139,11 @@ def allowed_mobile_prefixes_list(self):
             mobile_prefixes = self.allowed_mobile_prefixes.split(',')
         return mobile_prefixes
 
+    def get_sms_verification_enabled(self):
+        if self.sms_verification is None:
+            return app_settings.SMS_DEFAULT_VERIFICATION
+        return self.sms_verification
+
     def clean(self):
         if self.sms_verification and not self.sms_sender:
```

Comment:
This needs to be updated to use `get_sms_verification()` method.

Reply:
Are you suggesting to just return `self_verification` as it will never be `None`?

Reply:
I guess he meant `if self.get_sms_verification() and not self.sms_sender:`, right @pandafy?

Reply:
Yes, you are right @nemesisdesign 
```suggestion
        if self.get_sms_verification() and not self.sms_sender:
```